style.css

```diff
@@ -43,6 +45,196 @@
   display: block;
 }
 
+.expandcard {
+  display: none;
+  background-color: rgb(193, 199, 208, 0.5);
+  backdrop-filter: blur(5px);
+  position: fixed;
+  padding-top: 15px;
+  top: 0;
+  right: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.container {
+  width: 100%;
+  margin: 114px auto;
+  display: grid;
+  gap: 88px;
+}
+
+.card-body {
+  border-radius: 16px;
+  background-color: #fff;
+  padding: 16px;
+  width: 86%;
+  margin-top: 10%;
+  margin-bottom: 10%;
+}
+
+.card-body:hover {
+  animation-name: card-body;
+  animation-duration: 4s;
+}
+
+@keyframes card-body {
+  from {
+    background-color: #396df2;
+  }
+
+  to {
+    background-color: #b3bac4;
+  }
+}
+
+/* Popup Styling start */
+.model-body {
+  position: relative;
+  width: 320px;
+  height: 90%;
+  margin: 0 auto;
+  padding: 16px;
+  background-color: #fff;
+  border-radius: 16px;
+  z-index: 5;
+}
+
+#model-close {
+  position: absolute;
+  top: 30px;
+  right: 30px;
+}
+
+.line-break {
+  margin: 10px auto;
+  border: solid 1px #ebecf0;
+}
```

Comment:
- [ ] Hi @rmiyoyo, Great effort working on the pop up modal milestone of the portfolio project. I would like to highlight some of the things that requires attention below, Please kindly work on them

- On `Desktop and mobile view`.
    -  Kindly work on the button alignment and be sure they look exactly like what we have in the Figma design.
    - On class `.model-combine` kindly remove the `justify-content: space-around;` property.
    - and on the class `.model-button`, kindly reduce the `border-radius` to `8px` and add width of `45%` to it
    - lastly kindly give the button some `hover` and `active` effect as seen in Figma design.

- On `mobile view` alone
    - On class `.model-content > p`, kindly increase the `font-size` to `15px` so that it will look like what we have in the Figma design.

| Desktop View  | Mobile View |
| ------------- | ------------- |
| ![sap0](https://user-images.githubusercontent.com/78634841/233502392-d3a716e5-c10f-48a6-b8e8-3371c4c4a7d7.png) | ![sap1](https://user-images.githubusercontent.com/78634841/233502421-9ff81753-a217-4844-b8a7-6eded0806008.png) |